packages/esm-service-queues-app/src/queue-table/cells/queue-table-action-cell.component.tsx

```diff
@@ -9,6 +9,31 @@ export function QueueTableActionCell({ queueEntry }: QueueTableCellComponentProp
 
   return (
     <div>
+      {queueEntry.previousQueueEntry == null ? (
+        <Button
+          kind="ghost"
+          aria-label="Actions"
+          onClick={() => {
+            const dispose = showModal('void-queue-entry-modal', {
+              closeModal: () => dispose(),
+              queueEntry,
+            });
+          }}>
+          {t('void', 'Void')}
```

Comment:
"Void" isn't really an end-user facing term that we tend to use.  Typically, we'd present this as this "Delete".  Voiding is a technical detail as to how we perform deletes on the back end.